src/filter/filter_with_hvg/config.vsh.yaml

```diff
@@ -113,10 +116,10 @@ functionality:
     - path: ../../../resources_test/pbmc_1k_protein_v3
 platforms:
   - type: docker
-    image: python:3.8-slim
+    image: python:3.8
     python:
       packages:
-        - scanpy~=1.9.1
+        - git+https://github.com/DriesSchaumont/scanpy.git@013ae2dc4612b661ebeab0913a3e38b9fd13e3ab
```

Comment:
Is this custom version of scanpy still required?

Reply:
Yes, the PR is still open https://github.com/scverse/scanpy/pull/2302

Reply:
PR is merged, gj! Since scanpy is not released yet, shall we change this line to the scanpy repo itself?